### PR TITLE
Fix some minor errors

### PR DESCRIPTION
**Pull request by xujingjing-cmss**
Signed-off-by: xujingjing <xujingjing_yewu@cmss.chinamobile.com>

**What type of PR is this?**
/kind bug

**What this PR does / why we need it**:
some of the errors was not handled correctly.

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

### CODE REVIEW

File: keadm/cmd/keadm/app/cmd/debug/check.go [hunk 1]
```diff
@@ -285,7 +285,7 @@ func CheckNetWork(IP string, timeout int, cloudhubServer string, edgecoreServer
 	if config != "" {
 		edgeConfig, err := util.ParseEdgecoreConfig(config)
 		if err != nil {
-			err = fmt.Errorf("parse Edgecore config failed")
+			return fmt.Errorf("parse Edgecore config failed")
 		} else {
```
Comment (fisherxu) [inline keadm/cmd/keadm/app/cmd/debug/check.go:289]: You can delete the else here.

if err != nil {
			return fmt.Errorf("parse Edgecore config failed")
		} 

if cloudhubServer == "" {
				cloudhubServer = edgeConfig.Modules.EdgeHub.WebSocket.Server
......

Comment (xujingjing-cmss) [reply keadm/cmd/keadm/app/cmd/debug/check.go]: Sorry，I thought you ask me delete whole else part. Get it now.

Comment (fisherxu) [reply keadm/cmd/keadm/app/cmd/debug/check.go]: The code still need gofmted.

Comment (xujingjing-cmss) [reply keadm/cmd/keadm/app/cmd/debug/check.go]: the extra spaces have been cleaered

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**